R/env.R

```diff
@@ -704,17 +704,20 @@ with_bindings <- function(.expr, ..., .env = caller_env()) {
   .expr
 }
 
-#' Mask bindings by defining symbols deeper in a scope
+#' Mask symbols with a new child environment
+#'
+#' @description
+#'
+#' `env_mask()` is like [env_bind()] but it creates the bindings in a
```

Comment:
Should the prefix be `env_`? It doesn't actually work on environments, but objects that have environments?

Reply:
A lot of our `env_` functions are generic in this way. Do you want to reconsider this UI?

Reply:
I don't want to reconsider anything major; maybe it's just the framing here that threw me off? `env_mask()` now seems very similar to `child_env()`

Reply:
That's true. Probably best not change anything until we figure out `env` vs `child_env` then.

Reply:
I think the confusing thing about  `env_bury(x)` is that it returns the same type as `x`, not an environment. Do we have other functions that behave like that?

Reply:
I see `env_bind()` and variants, and `env_unbind()`.